Project.toml

```diff
@@ -2,14 +2,17 @@ name = "Polynomials"
 uuid = "f27b6e38-b328-58d1-80ce-0feddd5e7a45"
 license = "MIT"
 author = "JuliaMath"
-version = "0.5.3"
+version = "0.7.0"
```

Comment:
Yes, let's leave this as 0.7, add deprecations in 0.8, and then release 1.0 if there is no serious breakage or significant reservations expressed.